server.js

```diff
@@ -201,6 +201,20 @@ server.post('/invitation', jsonParser, function(req, res) {
   }
 })
 
+server.post('/activity_email', jsonParser, function(req, res) {    
+  var email = new sendgrid.Email({
+    to:       req.body.users,
+    from:     'wolontariat@krakow2016.com',
+    subject:  req.body.subject,
+    text:     req.body.text
+  })
+  sendgrid.send(email, function(err, json) {
+    console.log('sendgrid:', err, json)
+    res.send(err || json)
+  })  
+})
```

Comment:
To jest zły pomysł żeby dać możliwość komukolwiek, bez żadnej autoryzacji, wysyłania maili w naszym imieniu o dowolnej treści do kogokolwiek. To trzeba będzie przemyśleć i przepisać od początku.